examples/02_blog/index.php

```diff
@@ -11,10 +11,10 @@
 $schema = new BlogSchema();
 
 $processor = new Processor($schema);
-$payload = 'mutation { likePost(id:5) { title(truncated: false), status, likeCount } }';
-$payload = '{ latestPost { title, status, likeCount } }';
-$payload = '{ pageContentUnion { ... on Post { title, summary } ... on Banner { title, imageLink } } }';
-$payload = '{ pageContentInterface { title} }';
+// $payload = 'mutation { likePost(id:5) { title(truncated: false), status, likeCount } }';
+// $payload = '{ latestPost { title, status, likeCount } }';
+// $payload = '{ pageContentUnion { ... on Post { title, summary } ... on Banner { title, imageLink } } }';
+// $payload = '{ pageContentInterface { title} }';
```

Comment:
Why did you comment these out?

Reply:
Honestly, I cant recall

Reply:
Can you uncomment them so we can get this in?

Reply:
Now I recall, this is just examples, you can just run it with `php index.php`or similar. The $payload variable gets overwritten by the next line immediately, so you run just one at a time **anyway** :-)